src/pkg/library/client/push.go

```diff
@@ -130,16 +152,42 @@ func createContainer(name string, collectionID string) (id string, err error) {
 	return apiCreate(c, baseURL+"/v1/containers")
 }
 
-func createImage(name string, containerID string) (id string, err error) {
+func createImage(hash string, containerID string) (id string, err error) {
 	i := Image{
-		Name:        name,
+		Hash:        hash,
 		Description: "No description",
 		Container:   bson.ObjectIdHex(containerID),
 	}
 
 	return apiCreate(i, baseURL+"/v1/images")
 }
 
+func setTags(containerID string, imageID string, tags []string) error {
```

Comment:
Don't you need some documentation above functions or some such thing?

Reply:
The general rule is that public functions (starting with capitals) need doc comments. But that wouldn't preclude me adding some more here.